packages/pyright-internal/src/commands/commandResult.ts

```diff
@@ -0,0 +1,12 @@
+import { WorkspaceEdit } from 'vscode-languageserver-types';
```

Comment:
Please include a header comment and copyright notice.

Reply:
thanks